### PR TITLE
Remove `pvss` dependency

### PR DESCRIPTION
**Pull request by Jimbo4350**
#242

### CODE REVIEW

File: cardano-node/src/Cardano/CLI/Legacy/Byron.hs [hunk 1]
```diff
@@ -85,9 +63,9 @@ matchSize requestedSize lbl actualSize =
 -- | Encoder for a Byron/Classic signing key.
 --   Lifted from cardano-sl legacy codebase.
 encodeLegacyDelegateKey :: LegacyDelegateKey -> E.Encoding
-encodeLegacyDelegateKey dk@LegacyDelegateKey{lrkSigningKey=(SigningKey sk)}
+encodeLegacyDelegateKey (LegacyDelegateKey (SigningKey sk))
   =  E.encodeListLen 4
-  <> E.encodeListLen 1 <> encodeBinary (lrkVSSKeyPair dk)
+  <> E.encodeListLen 1 <> E.encodeNull
```
Comment (mrBliss) [inline cardano-node/src/Cardano/CLI/Legacy/Byron.hs]: Does this encoder need to backwards compatible with existing code? If so, I'm not sure it will be.

Comment (Jimbo4350) [reply cardano-node/src/Cardano/CLI/Legacy/Byron.hs]: @mrBliss If I understand correctly, I should use a dummy value instead of a null value here?

Comment (deepfire) [reply cardano-node/src/Cardano/CLI/Legacy/Byron.hs]: @Jimbo4350, we need to verify that key migration still works -- just take a delegate key from `cardano-sl` and feed it to the migration subcommand.

Comment (Jimbo4350) [reply cardano-node/src/Cardano/CLI/Legacy/Byron.hs]: @mrBliss @deepfire I can confirm key migration still works.

Comment (deepfire) [reply cardano-node/src/Cardano/CLI/Legacy/Byron.hs]: Thank you!

**Pull request merged**